Source/Dafny/Cloner.cs

```diff
@@ -976,6 +979,9 @@ class ScopeCloner : DeepModuleSignatureCloner {
 
     public override Function CloneFunction(Function f, string newName = null) {
       var basef = base.CloneFunction(f, newName);
+      basef.ByMethodTok = null; // never export the method body of a function-by-method
```

Comment:
The body is allowed to mention things that are private to the module. If these were exported as well, then importer would see things it should not. (In fact, the self-containedness checks of the export sets would fail.)

Reply:
Could you add a comment to the class `ScopeCloner` and consider making the name more verbose? I get the impression the class only clones parts of a module that are visible when the module is imported, but I didn't understand that from the name `ScopeCloner`.

Maybe a name like `ExportedSignatureCloner` would be more telling.